src/common/components/cards/how-to-fix-android-card-row.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { Term } from 'assessments/markup';
+import { FormattedResolution } from 'common/types/store-data/unified-data-interface';
+import { isEmpty, isNil } from 'lodash';
+import { CardRowProps } from '../../../common/configs/unified-result-property-configurations';
+import { NamedFC } from '../../../common/react/named-fc';
+import { SimpleCardRow } from './simple-card-row';
+
+export interface HowToFixAndroidCardRowProps extends CardRowProps {
+    propertyData: FormattedResolution;
+}
+
+export const HowToFixAndroidCardRow = NamedFC<HowToFixAndroidCardRowProps>('HowToFixAndroidCardRow', props => {
+    return (
+        <SimpleCardRow
+            label="How to fix"
+            content={<>{getHowToFixContent(props)}</>}
+            rowKey={`how-to-fix-row-${props.index}`}
+            contentClassName="how-to-fix-card-row"
+        />
+    );
+});
+
+type HowToFixSplit = {
+    str?: string;
+    match?: string;
+};
+
+function getHowToFixContent(props: HowToFixAndroidCardRowProps): (JSX.Element | string)[] {
+    const propertyData = props.propertyData;
+    let howToFixSplit: HowToFixSplit[] = [{ str: propertyData.howToFix }];
+    const result: (JSX.Element | string)[] = [];
+
+    if (!isEmpty(propertyData.formatAsCode)) {
+        propertyData.formatAsCode.forEach(item => {
+            howToFixSplit = getHowToFixSplitsForPattern(item, howToFixSplit);
+        });
+    }
+
+    howToFixSplit.forEach((item, index) => {
+        const key = `strong-how-to-fix-${props.index}-${index}`;
+        const content = isNil(item.str) ? item.match : item.str;
+
+        if (content[0] === ' ' && index !== 0) {
+            result.push(<span key={key + '-before-space'}>&nbsp;</span>);
+        }
+
+        result.push(isNil(item.str) ? <Term key={key}>{content}</Term> : <span key={key}>{content}</span>);
```

Comment:
![01 - isEmptry vs isNill result on a string](https://user-images.githubusercontent.com/2837582/66966177-2cf79e80-f031-11e9-8765-33ddfb43f0d9.png)
 are you sure you don't want to use `isEmpty` for checking this string values?